### PR TITLE
Selenium 1

### PR DESCRIPTION
**Pull request by andeemarks**
Hi folks,

I noticed you had an E2E acceptance card on your Trello at [https://trello.com/c/Pic0oq3Y/25-end-to-end-acceptance-testing](https://trello.com/c/Pic0oq3Y/25-end-to-end-acceptance-testing).

This branch contains a starting set of Selenium tests for the Customer and Admin sites.  I haven't done anything around routing yet, but basic adding of cargo is all covered.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -9,4 +9,4 @@ Development blog: https://citerus.github.io/dddsample-core/
 
 Trello board: https://trello.com/b/PTDFRyxd
 
-[![Build Status](https://travis-ci.org/citerus/dddsample-core.svg?branch=master)](https://travis-ci.org/citerus/dddsample-core)
+[![Build Status](https://travis-ci.org/andeemarks/dddsample-core.svg?branch=master)](https://travis-ci.org/andeemarks/dddsample-core)
```
Comment (jostly) [inline README.md]: This change shouldn't be merged

**Comment by jostly** [comment]
Good initiative! Please make sure you revert the change to the Readme that points to the Travis CI results.

**Comment by andeemarks** [comment]
Oops - thanks for spotting that Johan - I'll get to it today.

Cheers,
Andy

*Andy MARKS*
*Principal Consultant*
ThoughtWorks Melbourne
*mobile: *0435 318 616
*email*: amarks@thoughtworks.com
*web*: www.thoughtworks.com

**Comment by andeemarks** [comment]
README is now pointing back to the citerus Travis build

**Comment by andeemarks** [comment]
Hi Johan,

I've added some more the routing acceptance tests recently - would you like
me to commit these to the same branch to be merged as a single unit, or
rebase from cirterus/master and create another branch/PR for them?

Cheers,
Andy

*Andy MARKS*
*Principal Consultant*
ThoughtWorks Melbourne
*mobile: *0435 318 616
*email*: amarks@thoughtworks.com
*web*: www.thoughtworks.com

**Comment by mackos** [comment]
Converted your acceptance tests to use HtmlUnit and MockMvc instead of Selenium since the sample app now uses Spring Boot. Take a look in master.

**Pull request closed without merging**